client/src/ui/molecules/article-meta/index.tsx

```diff
@@ -0,0 +1,12 @@
+import "./index.scss";
+
+const ArticleMeta = ({ publishDate, readTime }) => {
```

Comment:
I don't know if this component is meant to stick around or if it's a quick hack to get the market research pages out. But I think we should stick to convention.
And if we're going to keep it, we should try to avoid `any` if we can for all new TS code. 
```suggestion
export function ArticleMeta({ publishDate, readTime }: {publishDate:string, readTime:string}) {
```

Reply:
Also, since this is never reused outside of `src/plus/deep-dives` I don't love that it's sitting in a global space (`ui`). 

It would have been better to put this into the article file itself.